cmds/tui.c

```diff
@@ -212,6 +217,7 @@ static void print_graph_total(struct field_data *fd)
 	d = node->time;
 
 	print_time(d);
+	field_width += 10;
```

Comment:
What about making these `print_*` functions to return width/characters it consumed like `printf()` does?

Reply:
Hmm.. It requires to modify all the print functions in `replay.c`, `graph.c` and `tui.c`. Do you prefer to modify all over the places?